packages/amplify-category-api/package.json

```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@aws-amplify/graphql-auth-transformer": "3.6.5",
-    "@aws-amplify/graphql-schema-generator": "0.9.4",
+    "@aws-amplify/graphql-schema-generator": "^0.9.4",
```

Comment:
I assume we're retaining this for backwards compat with our Gen 2 E2E tests? If so, I further assume we'll remove this dependency here and throughout once we migrate all those tests to CDK rather than CLI?

Reply:
Yes we will remove these dependencies and SQL E2Es once we migrate our tests to use CDK construct.